sdks/python/setup.py

```diff
@@ -25,19 +25,71 @@
 import sys
 import warnings
 from distutils.version import StrictVersion
+from distutils.errors import DistutilsError
 
 # Pylint and isort disagree here.
 # pylint: disable=ungrouped-imports
 import setuptools
 from pkg_resources import DistributionNotFound
 from pkg_resources import get_distribution
+from pkg_resources import normalize_path
+from pkg_resources import to_filename
+from setuptools import Command
 from setuptools.command.build_py import build_py
 from setuptools.command.develop import develop
 from setuptools.command.egg_info import egg_info
 from setuptools.command.sdist import sdist
 from setuptools.command.test import test
 
 
+class mypy(Command):
+  user_options = []
+
+  def initialize_options(self):
+    """Abstract method that is required to be overwritten"""
+
+  def finalize_options(self):
+    """Abstract method that is required to be overwritten"""
+
+  def get_project_path(self, include_dists=[]):
+      # with_2to3 = six.PY3 and getattr(self.distribution, 'use_2to3', False)
```

Comment:
leftover comments?

Reply:
yeah, this command class is based off of several in the distutils source, and it was unclear to me if we'd need to support this other case, so it's there as a reminder to figure that out. 

To be crystal clear, this PR is very much a WIP.  Just wanted to get eyes on it before I get too far ahead of myself.